app.py

```diff
@@ -23,10 +25,20 @@
 def static_file(path):
     return app.send_static_file(path)
 
+AUTH_TYPE = os.environ.get("AUTH_TYPE")
+
```

Comment:
```suggestion
# Initialize Azure keys based on authentication type and environment settings. 
# When AUTH_TYPE is not 'rbac' and USE_KEY_VAULT environment variable is set, 
# Azure keys are securely fetched from Azure Key Vault using DefaultAzureCredential. 
# Otherwise, keys are obtained from environment variables, with fallbacks to None or 
# an empty string depending on the AUTH_TYPE.

```